terraform/aws/analytical-platform-data-production/tooling-iam/transcribe-iam.tf

```
@@ -0,0 +1,54 @@
+data "aws_iam_policy_document" "transcribe_service_access" {
+  statement {
+    sid    = "transcribeListAccess"
+    effect = "Allow"
+    actions = [
+      "transcribe:List*"
+    ]
+    resources = ["*"]
+  }
+  statement {
+    sid    = "transcribeServiceAccess"
+    effect = "Allow"
+    actions = [
+      "transcribe:GetTranscriptionJob",
+      "transcribe:GetVocabulary",
+      "transcribe:CreateVocabulary",
+      "transcribe:StartTranscriptionJob",
+      "transcribe:StartStreamTranscription",
+      "transcribe:StartStreamTranscriptionWebSocket",
+      "transcribe:DeleteTranscriptionJob"
+    ]
+    resources = [
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:vocabulary/*",
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:transcription-job/*",
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:language-model/*",
+      "arn:aws:transcribe:eu-west-1:${var.account_ids["analytical-platform-data-production"]}:*"
+    ]
+  }
+}
+
+resource "aws_iam_policy" "transcribe_service_access" {
+  name   = "transcribe_service_access"
+  policy = data.aws_iam_policy_document.transcribe_service_access.json
+}
```

Normally we'd use a module for this

also:

```suggestion
resource "aws_iam_policy" "transcribe_service_access" {
  name   = "transcribe-service-access"
  policy = data.aws_iam_policy_document.transcribe_service_access.json
}
```